code/game/machinery/washing_machine.dm

```diff
@@ -59,15 +55,16 @@
 		M.Translate(rand(-3, 3), rand(-1, 3))
 		animate(src, transform=M, time=2)
 
-/obj/machinery/washing_machine/proc/clean_blood()
+/obj/machinery/washing_machine/clean_blood()
+	..()
 	if(!busy)
 		bloody_mess = FALSE
 		update_icon()
 
 /obj/machinery/washing_machine/proc/wash_cycle()
 	for(var/X in contents)
 		var/atom/movable/AM = X
-		SEND_SIGNAL(AM, COMSIG_COMPONENT_CLEAN_ACT, CLEAN_STRENGTH_BLOOD)

```

Comment:
Readd the clean_act signal, blood isn't the only thing that uses it

Reply:
Yes it is, actually. the only thing this signal did was clean_blood but with more obfuscation. I absolutely am 140% promising you that I tried every godamn way of preserving it but without the blood component, it didn't work.

Reply:
What is thermite?